apps/chat/src/store/prompts/prompts.reducers.ts

```diff
@@ -230,14 +236,19 @@ export const promptsSlice = createSlice({
       });
     },
     duplicatePrompt: (state, _action: PayloadAction<PromptInfo>) => state,
-    setPrompts: (state, { payload }: PayloadAction<{ prompts: Prompt[] }>) => {
-      state.prompts = payload.prompts;
+    setPrompts: (
+      state,
+      { payload }: PayloadAction<{ prompts: PromptInfo[] }>,
```

Comment:
add flag as here and update share epics for using it
https://github.com/epam/ai-dial-chat/pull/781/files#diff-be8fb500bbbc8a5659fc578fd9bc33f4c614b0a343a560b30c7457de12b203f7R348